src/index.ts

```diff
@@ -90,7 +90,8 @@ export const validatorCompiler: FastifySchemaCompiler<ZodAny> =
 export const serializerCompiler: FastifySerializerCompiler<ZodAny> =
   ({ schema }) =>
   (data) => {
-    const result = schema.safeParse(data);
+    // @ts-ignore
```

Comment:
❓

What's the type error here? If it is unavoidable, we should add a comment in code here to justify it for future readers.

Reply:
tried to tinker something up, please check